### PR TITLE
🔊🐛 [RUMF-201] add internal logs for buggy load event measures

### PR DESCRIPTION
**Pull request by BenoitZugmeyer**
To help debug the weird measure values shown in the View Explorer, this adds debug logs when the SDK sends abnormal load event duration (above one day). This is intentionaly at the lowest level possible (right before serialization) to make sure we have the final value transmited to the backend.

### CODE REVIEW

File: packages/core/src/transport.ts [hunk 1]
```diff
@@ -82,6 +84,20 @@ export class Batch<T> {
...
+    ) {
+      const loadEventEnd = contextualizedMessage.view.measures.load_event_end
+      if (typeof loadEventEnd === 'number' && loadEventEnd > 86400e9 /* 1 day in ns */) {
+        addMonitoringMessage(
+          `Buggy load event measure: ${loadEventEnd} - ` +
+            `View Id: ${contextualizedMessage.view.id} - ` +
+            `Document Version: ${contextualizedMessage.rum.document_version}`
+        )
+      }
+    }
```
Comment (bcaudan) [inline packages/core/src/transport.ts]: Can you extract this part in a dedicated function?

File: packages/core/src/transport.ts [hunk 2]
```diff
@@ -82,6 +84,20 @@ export class Batch<T> {
...
+      isObject(contextualizedMessage.view) &&
+      isObject(contextualizedMessage.view.measures) &&
+      isObject(contextualizedMessage.rum)
+    ) {
+      const loadEventEnd = contextualizedMessage.view.measures.load_event_end
+      if (typeof loadEventEnd === 'number' && loadEventEnd > 86400e9 /* 1 day in ns */) {
+        addMonitoringMessage(
+          `Buggy load event measure: ${loadEventEnd} - ` +
+            `View Id: ${contextualizedMessage.view.id} - ` +
+            `Document Version: ${contextualizedMessage.rum.document_version}`
```
Comment (bcaudan) [inline packages/core/src/transport.ts]: Do we not have all these elements already?

Comment (BenoitZugmeyer) [reply packages/core/src/transport.ts]: This is right before serialization, so we should have it

**Comment by BenoitZugmeyer** [comment]
> @bcaudan Do we not have all these elements already?

The purpose of this PR is to detect invalid values we may produce and log them along with the informations necessary to identify the associated View event.
With this, when filtering view with "load event > 1day" in the view explorer, we will be able to see if those were logged too.
* If they are, it means the issue is from the SDK side.
* If they are not, it means the issue is in the backend.

File: packages/core/src/transport.ts [hunk 3]
```diff
@@ -27,6 +27,25 @@ export class HttpRequest {
   }
 }
 
+const isObject = (o: unknown): o is { [k: string]: unknown } => typeof o === 'object' && o !== null
+
+function reportAdbnormalMeasure(contextualizedMessage: Context) {
```
Comment (bcaudan) [inline packages/core/src/transport.ts]:
```suggestion
function reportAbnormalMeasure(contextualizedMessage: Context) {
```

**Pull request merged**